src/convertToObject.js

```diff
@@ -3,16 +3,21 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';')
+    .filter((item) => item.trim() !== '')
+    .map((item) => item.split(':'))
+    .reduce((acc, item) => {
+      return {
+        ...acc,
+        [item[0].trim()]: item[1].trim(),
+      };
```

Comment:
```suggestion
    .reduce((acc, [key, value]) => {
      return {
        ...acc,
        [key.trim()]: value.trim(),
      };
```
You can use destructuring.